src/cli/subcommands/fetch_params_cmd.rs

```diff
@@ -51,14 +51,14 @@ fn ram_to_int(size: &str) -> anyhow::Result<SectorSize> {
     let mut trimmed = size.trim_end_matches('B');
     trimmed = trimmed.trim_end_matches('b');
 
-    type SectorSize = <crate::shim::sector::SectorSize as Inner>::FVM;
+    type SectorSize = crate::shim::sector::SectorSize;
 
     match trimmed {
-        "2048" | "2Ki" | "2ki" => Ok(SectorSize::_2KiB.into()),
-        "8388608" | "8Mi" | "8mi" => Ok(SectorSize::_8MiB.into()),
-        "536870912" | "512Mi" | "512mi" => Ok(SectorSize::_512MiB.into()),
-        "34359738368" | "32Gi" | "32gi" => Ok(SectorSize::_32GiB.into()),
-        "68719476736" | "64Gi" | "64gi" => Ok(SectorSize::_64GiB.into()),
+        "2048" | "2Ki" | "2ki" => Ok(SectorSize::_2KiB),
```

Comment:
Can't help but wonder if we need to support 3 representations of each sector size. At the very least we could ignore the case and have it `to_lower`.